cot/src/openapi/swagger_ui.rs

```diff
@@ -94,10 +96,8 @@ impl App for SwaggerUi {
     fn router(&self) -> Router {
         let swagger_ui = Arc::new(self.inner.clone());
         let swagger_handler = async move || {
-            Ok(Response::new_html(
-                StatusCode::OK,
-                Body::fixed(swagger_ui.serve().map_err(cot::Error::custom)?),
-            ))
+            let swagger = swagger_ui.serve().map_err(cot::Error::custom)?;
+            Body::fixed(swagger).with_content_type(HTML_CONTENT_TYPE)
```

Comment:
```suggestion
            Ok::<_, cot::Error>(Html::new(swagger))
```

If we have fancy new types, why not use them?